shopinvader_product_attribute_set/schemas/product.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from typing import Any
+
+from odoo.addons.shopinvader_product.schemas import ProductProduct as ProductProductBase
+
+from . import ProductAttributeSet
+
+
+class ProductProduct(ProductProductBase, extends=True):
+    attribute_set: ProductAttributeSet | None = None
+    attributes: dict[str, Any] = {}
+    structured_attributes: list[dict[str, Any]] = [{}]
+
+    @classmethod
+    def from_product_product(cls, odoo_rec):
+        obj = super(ProductProduct, cls).from_product_product(odoo_rec)
```

Comment:
```suggestion
        obj = super().from_product_product(odoo_rec)
```

Reply:
I am getting a strange error if I remove the class and cls from super

```
    @classmethod
    def from_product_product(cls, odoo_rec):
>       obj = super().from_product_product(odoo_rec)
E       TypeError: super(type, obj): obj must be an instance or subtype of type
```

Reply:
@sbejaoui It's because your schema module is not loaded when the module is loaded by odoo and the extendable registry not properly initialized into your tests... see https://github.com/shopinvader/odoo-shopinvader/pull/1450/commits/85ba911e6a75b8a13b360003f73708e69cec19e8